src/components/dashboard/FaceRecognition/ZoomCapture.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
+import { Section } from '../../common'
+import type { DashboardProps } from '../Dashboard'
+import logger from '../../../lib/logger/pino-logger'
+import { Camera, getResponsiveVideoDimensions } from './Camera.web'
+import { capture, type ZoomCaptureResult } from './Zoom'
+
+const log = logger.child({ from: 'ZoomCapture' })
+type ZoomCaptureProps = DashboardProps & {
+  screenProps: any,
+  loadedZoom: Boolean,
+  showZoomCapture: Boolean,
+  height: number,
+  width: number,
+  onCaptureResult: (captureResult?: ZoomCaptureResult) => void,
+  onError: (error: string) => void
+}
+
+/**
+ * Responsible for Zoom client SDK triggering:
+ * 1. Calls zoom.capture() on the camera capture (Recieved from Camera component)
+ * 2. Triggers callback when captureResult is ready
+ */
+class ZoomCapture extends React.Component<ZoomCaptureProps> {
+  videoTrack: MediaStreamTrack
+
+  captureUserMediaZoom = async (track: MediaStreamTrack) => {
+    this.videoTrack = track
+    let captureOutcome: ZoomCaptureResult
+    try {
+      log.debug('zoom performs capture..')
+      let zoomSDK = this.props.loadedZoom
+      captureOutcome = await capture(zoomSDK, track) // TODO: handle capture errors.
+    } catch (e) {
+      log.error(`Failed on capture, error: ${e}`)
+    }
+    log.info({ captureOutcome })
+    this.props.onCaptureResult(captureOutcome)
+  }
+
+  componentWillUnmount() {
+    log.debug('Unloading video track?', !!this.videoTrack)
+    this.videoTrack && this.videoTrack.stop()
```

Comment:
this is done in camera component

Reply:
removed from ZoomCapture on latest commit b60cbe2f19564502b53b76db5614628051c384b7